packages/phone-number-privacy/signer/src/common/metrics.ts

```diff
@@ -22,8 +22,8 @@ export const Counters = {
     labelNames: ['endpoint', 'statusCode'],
   }),
   databaseErrors: new Counter({
-    name: 'database_read_errors',
```

Comment:
Note: We messed up and didn't change the name of this when we made it a counter for both read and write errors. We'll have to update this dashboard to account for this change.

Reply:
Added a TODO in this ticket:
https://app.zenhub.com/workspaces/cap-sprint-board-60059934ed9c6f000e5594f6/issues/celo-org/celo-monorepo/7798